src/sdk/noobaa_s3_client/noobaa_s3_client.js

```diff
@@ -0,0 +1,238 @@
+/* Copyright (C) 2023 NooBaa */
+'use strict';
+
+const { S3ClientSDKV2 } = require('./noobaa_s3_client_sdkv2');
+const url = require('url');
+const { S3 } = require("@aws-sdk/client-s3");
+const { NodeHttpHandler } = require("@aws-sdk/node-http-handler");
+const dbg = require('../../util/debug_module')(__filename);
+dbg.set_process_name('noobaa_s3_client_by_sdk');
+const config = require('../../../config');
+const http_utils = require('../../util/http_utils');
+
+// The params are the AWS SDK V2 params.
+// params = a map of parameters that are passed to the constructor of S3 Object in order to to bind to every request
+function get_s3_client_v2_params(params) {
```

Comment:
redundant. we can change the params to v3

Reply:
Removed, as discussed we will change hard-coded the params to v3 structure.